front-end/src/app/interview/edit-interview/edit-interview.component.html

```diff
@@ -29,37 +21,48 @@
-              {{status?.name}}
-            </option>
-          </select>
+        <div class="form-group row mb-0 mt-4">
+          <div class="col-sm-4 offset-sm-2">
+            <label class="text-label">Interview</label>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-sm-5 offset-sm-2">
+            <select class="custom-select" name="statusCandidate" id="statusCandidate" formControlName="statusCandidate">
+              <option [ngValue]="status.id" *ngFor="let status of statusCandidateList">
+                {{status?.name}}
+              </option>
+            </select>
+          </div>
+        </div>
+        <!--COMMENT-->
+        <div class="form-group row mb-0 mt-4">
+          <div class="col-sm-3 offset-sm-2">
+            <label class="text-label" for="comment">Comment</label>
+          </div>
         </div>
-
-        <!-- COMMENT -->
-        <div class="form-group">
-          <label class="font-weight-bold" for="comment">Comment</label>
-          <textarea class="form-control" id="comment" formControlName="comment" rows="3"
+        <div class="row">
+          <div class="col-sm-8 offset-sm-2">
+            <textarea class="form-control" id="comment" formControlName="comment" rows="10" cols="12"
             [ngClass]="{ 'is-invalid': submitted && f.comment.errors }" placeholder="Write your comment here..."
             [maxlength]="300"></textarea>
-          <div *ngIf="submitted && f.comment.errors" class="invalid-feedback margin-left">
-            <div *ngIf="submitted && f.comment.errors?.maxlength">Comment must have less than 300 characters</div>
-            <div id="only-letter-comment" *ngIf="submitted && f.comment.errors?.pattern">Comment has to be only
+            <div *ngIf="submitted && f.comment.errors" class="invalid-feedback margin-left">
+              <div *ngIf="submitted && f.comment.errors?.maxlength">Comment must have less than 300 characters</div>
+              <div id="only-letter-comment" *ngIf="submitted && f.comment.errors?.pattern">Comment has to be only
               characters</div>
+            </div>
+          </div>
+        </div>          
+        <!--PARTICIPANTS-->
+        <div class="form-group row mb-0 mt-4">
+          <div class="col-sm-4 offset-sm-2">
+            <label for="participants" class="text-label">Participants</label>
           </div>
         </div>
-
-        <!-- ADD PARTICIPANT -->
-        <div class="form-group">
-          <label for="participants" class="font-weight-bold">Participants</label>
-
-          <div class="row">
-            <div class="col-12 col-md-9 mb-3">
-              <input id="participant" type="text" formControlName="addParticipant" placeholder="Add Participants"
+        <div class="row">
+          <div class="col-sm-5 offset-sm-2">
+            <input id="participant" type="text" formControlName="addParticipant" placeholder="Add Participants"
```

Comment:
- When i hover the "Add Participants" button with the mouse it looks like something in the page resize, im testing this in chrome 

- To me, the trash can icon looks too big and adds an unwanted vertical space, but this is a subjective point of view if you think it's better like it is now it is fine too.o 👍 

![image](https://user-images.githubusercontent.com/23131424/77769825-c08abd80-701a-11ea-9c69-5448b49ac94c.png)

Reply:
> * To me, the trash can icon looks too big and adds an unwanted vertical space, but this is a subjective point of view if you think it's better like it is now it is fine too.o

Thank you for your suggestions :ok_hand:. I made the corresponding improvements to the buttons, in addition to reducing the size of the garbage icon.

Reply:
> * When i hover the "Add Participants" button with the mouse it looks like something in the page resize, im testing this in chrome
> * To me, the trash can icon looks too big and adds an unwanted vertical space, but this is a subjective point of view if you think it's better like it is now it is fine too.o +1
> 
> ![image](https://user-images.githubusercontent.com/23131424/77769825-c08abd80-701a-11ea-9c69-5448b49ac94c.png)

Thank you for your suggestions :ok_hand:
I made the corresponding improvements to the buttons, in addition to reducing the size of the garbage icon.